katacomb/katacomb/util/__init__.py

```diff
@@ -651,7 +651,7 @@ def selection_options(parser):
                        help="Range of channels to image, must be of the form <start>,<end>. "
                             "Default: Image all (unmasked) channels.")
     group.add_argument("--pols",
-                       default="HH,VV",
+                       default="HH,VV,HV,VH",
```

Comment:
I'm not sure why you are changing the defaults here ? Won't this change the why the online pipeline selects data ?

Reply:
I struggled back and forth on this one and decided that just for UV extraction, full pol would be the default - but this means the offline imager also gets full pol and we don't image anything other than Stokes I by default. Thinking about it some more I think you are right and I'll change it back to `HH,VV` since it doubles the read time from the archive for offline mode, also `mvftoms.py` only does HH,VV by default.

The data selection arguments are not used by the online pipeline so this option will have no effect there. Data selection for that is done directly via a katdal selection statement (the `--select` script argument). That katdal selection is passed to the script by the master controller [here](https://github.com/ska-sa/katsdpcontroller/blob/master/src/katsdpcontroller/generator.py#L2723). It might be a little surprising, but currently the online pipeline reads full pol - the current imager default is just to ignore HV,VH data for Stokes I. This has never really bothered me that much since the online data transfer is fast enough - but we could switch it off in the master controller and save a bit of time.

Reply:
That sounds good, we'll keep HH, VV as the default for offline use and leave the online system as it is as there is no compelling reason to change it :)